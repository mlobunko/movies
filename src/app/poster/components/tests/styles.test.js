import React from 'react';
import { shallow } from 'enzyme';
import 'enzyme-to-json';

import {
  StyledPoster,
  StyledPosterContainer,
  StyledPosterContent
} from '../styles';

const posterPath = 'abc';

describe('StyledPoster', () => {
  it('renders correctly, posterPath true', () => {
    const wrapper = shallow(<StyledPoster posterPath={posterPath} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly, posterPath false', () => {
    const wrapper = shallow(<StyledPoster posterPath={''} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledPosterContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledPosterContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledPosterContent', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledPosterContent />);
    expect(wrapper).toMatchSnapshot();
  });
});
