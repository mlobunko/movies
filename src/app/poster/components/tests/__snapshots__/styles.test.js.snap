// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledPoster renders correctly 1`] = `
<div
  className="sc-bdVaJa dLEHil"
/>
`;

exports[`StyledPosterContainer renders correctly 1`] = `
<div
  className="sc-bwzfXH cAyBcM"
/>
`;

exports[`StyledPosterContent renders correctly 1`] = `
<div
  className="sc-htpNat jIWaBW"
/>
`;
